src/hidpp20.h

```diff
@@ -317,6 +324,46 @@ union hidpp20_button_binding {
 } __attribute__((packed));
 _Static_assert(sizeof(union hidpp20_button_binding) == 4, "Invalid size");
 
+struct hidpp20_color {
+	uint8_t red;
+	uint8_t green;
+	uint8_t blue;
+};
+
+// TODO: put the leds into array and access them through this enum
+enum hidpp20_led_type {
+	HIDPP20_LED_UNKNOWN = -1,
+	HIDPP20_LED_LOGO = 0,
+	HIDPP20_LED_SIDE
+};
+
+struct hidpp20_internal_led {
+	uint8_t mode;                  /* 00 - off; 01 - on; 03 - cycle; 0a - breath */
+	struct hidpp20_color color;
+	union {
+		struct {
+			uint8_t free[2];
+			uint16_t rate;          /* 100 - 20000 */
```

Comment:
logitech drivers sends between 16000 - 20000 values, 16000 being 1 sec cycle and 20000 being 20 sec cycle,
i have no idea what kind of unit it is.